src-plugins/itkDataImage/writers/itkDicomDataImageWriter.cpp

```diff
@@ -0,0 +1,492 @@
+/*=========================================================================
+
+ medInria
+
+ Copyright (c) INRIA 2013 - 2014. All rights reserved.
+ See LICENSE.txt for details.
+
+  This software is distributed WITHOUT ANY WARRANTY; without even
+  the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR
+  PURPOSE.
+
+=========================================================================*/
+
+#include <itkDicomDataImageWriter.h>
+
+#include <medAbstractData.h>
+#include <medAbstractDataFactory.h>
+
+
+#include <itkExtractImageFilter.h>
+#include <itkGDCMImageIO.h>
+#include <itkImage.h>
+#include <itkImageFileWriter.h>
+#include <itkMetaDataObject.h>
+
+#include <gdcmUIDGenerator.h>
+
+#include <string>
+
+static QString s_identifier() {
+    return "itkDicomDataImageWriter";
+}
+
+static QStringList s_handled() {
+    return QStringList()  << "itkDataImageChar3"
+                          << "itkDataImageUChar3"
+                          << "itkDataImageShort3"
+                          << "itkDataImageUShort3"
+                          << "itkDataImageInt3"
+                          << "itkDataImageUInt3"
+                          << "itkDataImageLong3"
+                          << "itkDataImageULong3"
+                          << "itkDataImageFloat3"
+                          << "itkDataImageDouble3";
+}
+
+itkDicomDataImageWriter::itkDicomDataImageWriter(): itkDataImageWriterBase() {
+    this->io = itk::GDCMImageIO::New();
+}
+
+itkDicomDataImageWriter::~itkDicomDataImageWriter()
```

Comment:
this function is not used

Reply:
I added it to stay consistent with the others writer